Battleship/Salvo.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+
+namespace Battleship
+{
+    public class Salvo
+    {
+        private Player Opponent;
+        private List<Coordinate> Shots;
+        public Salvo(Player opponent, List<Coordinate> shots)
+        {
+            Opponent = opponent;
+            Shots = shots;
+        }
+
+        public List<Coordinate> Results(){
+            List<Coordinate> hits = new List<Coordinate>();
+            foreach (var coordinate in Shots)
+            {
+                foreach (Ship ship in Opponent.Fleet.Ships)
+                {
+                    foreach(Coordinate cor in ship.Position.Coordinates){
+                        if(cor == coordinate){
+                            ship.ShipDamage = new bool[ship.ShipDamage.Length-1];
+                            hits.Add(cor);
+                        }
+                    }
+                }
+            }
+            return hits;
+        }
```

Comment:
We should probably track and display misses, too.

Reply:
Right. I hadn't decided yet how best to display results. I played with the idea of a "player response" class that would have hits, misses, and "battleship sunk". But I am still trying to figure out where to put central control logic.

Reply:
Maybe hits and misses. But whether a ship is sunk could be in a separate method.